egs/aidatatang_200zh/ASR/pruned_transducer_stateless2/decoder.py

```diff
@@ -0,0 +1,103 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang)
```

Comment:
please replace such files with symlinks.

Reply:
Done and tested.